open_spiel/python/algorithms/nash_averaging.py

```diff
@@ -44,13 +44,85 @@ def _max_entropy_symmetric_nash(p_mat, eps=1e-9):
   n = len(p_mat)
   x = cp.Variable(shape=n)
   obj = cp.Maximize(cp.sum(cp.entr(x)))
-  a_mat = np.ones(n).reshape((1, n))
-  constraints = [p_mat @ x <= 0, a_mat @ x == 1, x >= eps * np.ones(n)]
+  constraints = [p_mat @ x <= 0, x >= eps * np.ones(n)]
```

Comment:
Why can't the nonnegative x constraints be specified with a 0?

Reply:
here eps is a parameters that bounds the minimum probability mass. if eps=0 then it becomes negative constraints x>=0